src/plugins/dashboard/public/dashboard_constants.ts

```diff
@@ -77,7 +77,7 @@ export const DASHBOARD_CACHE_TTL = 1000 * 60 * 5; // time to live = 5 minutes
 // Default State
 // ------------------------------------------------------------------
 export const DEFAULT_DASHBOARD_INPUT: Omit<DashboardContainerInput, 'id'> = {
-  viewMode: ViewMode.EDIT, // new dashboards start in  edit mode.
+  viewMode: ViewMode.VIEW,
```

Comment:
I'm a bit confused on why view is now the default? 🤔

Reply:
I just changed it to be a little more **default**. Basically, this used to be the fallback for newly created Dashboards, but now it should _always_ get overwritten with something, so it made more sense to have the default input contain view mode.

Reply:
Makes sense to me :+1: Thanks!